janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/utils/CassandraHelper.java

```diff
@@ -80,7 +80,7 @@ private FilterResultColumns(StaticBuffer lastColumn, int limit,
 
         @Override
         public boolean apply(@Nullable E e) {
```

Comment:
We could use the https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html from jetbrains. Here we write that is nullable but make a check that is not null.